CMakeLists.txt

```diff
@@ -14,6 +14,14 @@ if("${CMAKE_BUILD_TYPE}" STREQUAL "")
 	set(CMAKE_CXX_FLAGS_RELWITHDEBINFO "-Wall -g -O2 -fPIC")
 endif()
 
+if(DEFINED BUILD_PORTABLE_LINUX)
```

Comment:
Portable linux build. I'm not entirely sure what to do about libsodium -- right now, it's linked statically here. As such, we'd have to roll out a new version whenever sodium has a relevant patch...